src/libs/API/parameters/CategorizeTrackedExpenseParams.ts

```diff
@@ -16,7 +17,7 @@ type CategorizeTrackedExpenseParams = {
     reportPreviewReportActionID: string;
     category?: string;
     tag?: string;
-    receipt?: Receipt;
+    receipt: OnyxEntry<Receipt>;
```

Comment:
```suggestion
    receipt: Receipt | null;
```

`Receipt` is not a value stored in Onyx